test/start-typescript.test.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const util = require('util')
+const { once } = require('events')
+const fs = require('fs')
+const crypto = require('crypto')
+const baseFilename = `${__dirname}/fixtures/test_${crypto.randomBytes(16).toString('hex')}`
+
+const t = require('tap')
+const test = t.test
+const sgetOriginal = require('simple-get').concat
+const sget = (opts, cb) => {
+  return new Promise((resolve, reject) => {
```

Comment:
please use promisify instead.

Reply:
I used promisify.custom because, I think, it is not possible to simply use promisify if the callback has more than one param plus the err, in this case `(err, response, body)`.
Am I wrong?